CoolCurrency/Controller/MatchCurrencyGameViewController.swift

```diff
@@ -10,17 +10,36 @@ import UIKit
 class MatchCurrencyGameViewController: UIViewController, UIPickerViewDelegate, UIPickerViewDataSource {
     
     @IBOutlet private weak var matchCurrencyPickerView: UIPickerView!
+    @IBOutlet private weak var finalTitle: UILabel!
+    @IBOutlet private weak var finalScore: UILabel!
+    @IBOutlet private weak var playAgain: UIButton!
+    @IBOutlet private weak var matchSymbolToFlag: UIButton!
+    @IBOutlet private weak var scoreBoard: UIView!
     
     private lazy var viewModel = MatchCurrencyGameViewModel()
     
     override func viewDidLoad() {
         super.viewDidLoad()
+        setLabelsHidden(true)
         matchCurrencyPickerView.delegate = self
         matchCurrencyPickerView.dataSource = self
     }
     
-    @IBAction func matchButtonPressed(_ sender: UIButton) {
+    @IBAction private func matchButtonPressed(_ sender: UIButton) {
         showUserSuccessAlert(viewModel.checkIfCorrect())
+        if viewModel.shouldDisplayAnswer() == true {
+            setLabelsHidden(false)
```

Comment:
You could actually drop this if statement if you wanted and do something like this: 

```
setLabelsHidden(!viewModel.shouldDisplayAnswer())
setButtonsHidden(viewModel.shouldDisplayAnswer())
```

Reply:
So I had to declare a boolean then use it as my parameter because if I use setLabelsHidden(!viewModel.shouldDisplayAnswer())
setButtonsHidden(viewModel.shouldDisplayAnswer()), it will call viewModel.shouldDisplayAnswer() twice meaning it will check for the answer twice doubling the points for every question